client/my-sites/site-settings/navigation-back-button/README.md

```diff
@@ -0,0 +1,41 @@
+*Navigation Back Button*
+
+This component is used to display a `Back` button for redirection to the previously
+visited site.
+
+#Usage:
+
+```js
+import NavigationBackButton from 'my-sites/site-settings/navigation-back-button';
+...
+
+getRoute() {
+	... // return a route
+}
+
+<span className="....">
```

Comment:
Would you like to place this within a `render()` method, just so it's clear where one's supposed to be calling this one? Also, maybe you want to remove `{  ...this.props }` from the example.